doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,137 @@
+
+# PaddlePaddle Client
+PaddlePaddle client is command line tool, you can use a PaddlePaddle client to start a local train and submit a distributed training job to kubernetes cluster.
+
+The relation of PaddlePaddle, kubernetes and docker:
+
+<img src="./submit-job.png" width="500">
```

Comment:
Questions with this figure:

1. I am not sure that pservers and trainers should be in two jobs. In our current configuration, each trainer has a pserver running on the same physical node to optimally overlay networking and computing.

1. `paddle` must communicate with Kubernetes' API server to start the job. It might communicate to the master process of the job too.